### ISSUE TITLE
Continious deployment?

### ISSUE THREAD

**Issue by Guppster** [ArkEcosystem/ark-java#25]
There has been some significant development in this repository but the bintray maven repo is never updated. Can we maybe have Travis setup to run the tests and push the latest version of Master to bintray?.
https://bintray.com/arkecosystem/ark-java/client/0.3#

**Comment by dafty** [comment]
@Guppster I've created a [travis-ci](https://github.com/ArkEcosystem/ark-java/tree/travis-ci) branch and added the minimal config to there, feel free to play around.

In the near future, I think we should prepare a 1.0.0 branch (new semantic versioning) with the new tests using stubs, automated Travis builds, and anything new that might come up in the meantime. This would be released as the first major version.

FWICS, master has been on 0.4 now for some time, but the artifacts were not pushed to remote.

Since a few issues on 0.4 have now been fixed locally and the tests we have are green on travis-ci, I would say the current version is stable.

What are your thoughts on incrementing to 0.5 and pushing to remote now?

After 0.5 is built, we can start work on 1.0.0, release it, and end up with a clean versioning system to start from.
